decryptor/postgresql/packet_handler.go

```diff
@@ -447,15 +458,45 @@ func (packet *PacketHandler) ReadClientPacket() error {
 		We read first 5 bytes, check is there known message types. If not then we try to read 3 more bytes and recognize 3 special messages
 		by their values. If we don't recognize, then process it as general message. We may have error if it's unknown message with message length < 8 bytes when
 		we not recognize, try to read +3 bytes and will block on system call read to read more when message may have only 1 bytes of MessageType and minimal MessageLength = 4 bytes (itself)
+		Overall, as of today (PostgreSQL 14), the protocol supports following packets, that can be
```

Comment:
will be great to update whole comment block) because above we mention 3 packets)